src/System.Windows.Forms/src/System/Windows/Forms/Control.cs

```diff
@@ -8434,139 +8454,127 @@ private static void PaintBackColor(PaintEventArgs e, Rectangle rectangle, Color
 
             // Note: PaintEvent.HDC == 0 if GDI+ has used the HDC -- it wouldn't be safe for us
             // to use it without enough bookkeeping to negate any performance gain of using GDI.
-            if (color.A == 255)
+            if (color.HasTransparency())
```

Comment:
I don't think it is equivalent, `A==255` is opaque
```suggestion
            if (!color.HasTransparency())
```

Reply:
Whoops, good catch :)